addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties

```diff
@@ -4,7 +4,7 @@ paramminer.error.nofile = File not found : {0}
 paramminer.panel.msg = <html>Param Miner panel</html>
 paramminer.panel.title = Param Miner
 paramminer.popup.msg = Example right click menu invoked for:\n {0}
-paramminer.popup.title = Param Miner: Example right click menu
```

Comment:
You could change this to something more relevant.
Thinking about it ... this option should probably go under the "Attack" menu.
Not a problem for now, just something to add to your todo list ;)

Reply:
Actually, I thought of having separate submenus for it :) with guess and "attack" sections later on.